lib/index.js

```diff
@@ -46,187 +43,103 @@ Ramen.prototype.initialize = function() {
  */
 
 Ramen.prototype.loaded = function() {
-  return is.object(window.Ramen);
+  return !!(window._ramen && window._ramen.push !== Array.prototype.push);
 };
 
-/**
- * Identify.
- *
- * @api public
- * @param {Identify} identify
- */
+Ramen.prototype.group = function(group) {
+  if (!this.identified) {
+    this.identify({ anonymous: true });
+  }
 
-Ramen.prototype.identify = function(identify) {
-  privateGo({ integration: this, identify: identify });
-};
+  var props = group.properties();
```

Comment:
Can you change this to `group.traits()` since we use `traits` for group calls. Although functionally speaking this is okay since `.properties()` [looks up traits](https://github.com/segmentio/facade/blob/master/lib/group.js#L124) underneath anyway but just for the sake of keeping consistency across integrations :)